README.md

```diff
@@ -1,25 +1,75 @@
-# must-gather for Red Hat OpenShift Data Science
+# must-gather for Red Hat OpenShift AI
 
 The must-gather script allows a cluster admin to collect information about various key resources and namespaces
 for Red Hat OpenShift Data Science.
```

Comment:
"Red Hat OpenShift Data Science" -> "Red Hat OpenShift AI"

Reply:
good catch! i will update it , thanks!